src/components/Sum.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
+    {' '}
```

Comment:
There is much easier way to render this text:
```suggestion
{`Sum of ${a} and ${b} is ${a + b}`}
```